src/smefit/loader.py

```diff
@@ -191,8 +193,9 @@
         # here return both exp sys and t0 modified sys
         return central_values, df, df_t0, stat_error, luminosity
 
+    @staticmethod
```

Comment:
Because I wanted to call it in load_scales... The main thing was that it seems like a method that it makes sense to be able to call without creating an instance of the object

Reply:
Ah I see now! Makes sense, this method doesn't use any other attributes really so I'm fine with keeping it static.